docs/helpers/RemoteDeviceHelper.md

```diff
@@ -0,0 +1,102 @@
+---
+title: RemoteDeviceHelper
+author: avknaidu
+description: The RemoteDeviceHelper is a Helper to get a List Remote Devices that are accessible.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, RemoteDeviceHelper, helper
+dev_langs:
+  - csharp
+---
+
+# RemoteDeviceHelper 
+
+The [RemoteDeviceHelper](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.helpers.remotedevicehelper) gives you a list of Remote Systems. All the systems must be signed in with the same Microsoft Account (MSA)
+
+> [!IMPORTANT]
+> Make sure you enable "RemoteSystem" Capability in `package.appxmanifest`
```

Comment:
Can we keep the capability doc link here to be consistent with our CameraPreview format for this message?
```suggestion
> Make sure you enable the [RemoteSystem capability](https://docs.microsoft.com/en-us/windows/uwp/packaging/app-capability-declarations#general-use-capabilities) in your app's `package.appxmanifest` to access remote system information.
```